docs/configuration/authentication/digid.rst

```diff
@@ -32,85 +69,102 @@ be :ref:`prefilled <configuration_prefill_index>` with relevant personal data.
+.. _configuration_authentication_digid_prepare_certificates:
 
-#. **Key pair**: Upload the `PKIoverheid Private Services Server G1`_ and the private key (see point 2 of the next section).
-   The certificate will be used in the metadata that will be generated, and the private key will be used to sign the metadata.
+Preparing the certificates
+==========================
 
-#. **Metadata file(XML) URL**: This is the URL where the metadata (XML) for the Identity Provider can be obtained.
-   The entity ID (of the identity provider) will automatically be extracted from the XML.
+To request a *PKIoverheid certificate*, you need a private key and a certificate signing
+request. You can generate these using ``openssl`` or other utilities, or use the built-in
+signing requests in the admin interface.
 
-#. **Entity ID**: This is the entity ID of Open-Forms. For DigiD, this can be the URL where Open-Forms is deployed, for
-   example ``https://openforms.test.nl``.
+.. tip:: Creating the signing requests from the admin ensures the private key never
+   leaves the server, lessening the chances that it is accidentally leaked.
 
-#. **Base URL**: This is the URL where Open-Forms is deployed.
+**Using the admin interface**
 
-#. **Resolve artifact binding content type**: This is the the value of the ``Content-Type`` header for the resolve artifact binding request.
-   Modern brokers typically expect ``text/xml`` while ``application/soap+xml`` is considered legacy.
+#. Navigate to the **Admin**.
+#. In the menu, navigate to **Configuration** > **Signing requests**.
+#. In the top right of the page, click the **Add signing request** button.
+#. Enter the desired fields - the information will be included in the signing request.
+   Consult with your certificate supplier which data is required and what values are
+   expected.
+#. Click **Save** to persist the changes. The page is reloaded.
+#. Under the section *Signing request (CSR)*, click the **Download CSR** link.
 
-#. **Want assertions signed**: This should be **checked**.
+Now, send the downloaded CSR to your certificate supplier and wait for them to verify
+it and provide you with the matching certificate. This can take some time (hours to
+days), as verification is often a manual process.
 
-#. **Signature algorithm**: Select ``RSA_SHA1``.
+Once you have received the certificate, navigate back to the admin and locate your
+signing request.
 
-#. **Digest algorithm**: Select ``SHA1``.
+#. Edit the original signing request record in the admin.
+#. Navigate to the *Upload signed certificate* section.
+#. Upload the certificate provided by the certificate supplier.
+#. Save the changes.
 
-#. **Service name**: This is the name of the service for which authentication is needed.
+If everything is valid, the private key + public cerificate pair is now available and
+almost ready for use. You can verify it in the admin via **Configuration** >
```

Comment:
almost? :sweat_smile:

Reply:
yeah, you still need to link it to either digid or eherkenning in the digid/eherkenning django app (extra model there)